monkey/infection_monkey/exploit/powershell_utils/powershell_client.py

```diff
@@ -42,14 +43,16 @@ def format_password(credentials: Credentials) -> Optional[str]:
     if credentials.secret_type == SecretType.CACHED:
         return None
 
+    plaintext_secret = get_plain_text(credentials.secret)
+
     if credentials.secret_type == SecretType.PASSWORD:
-        return credentials.secret
+        return plaintext_secret
 
     if credentials.secret_type == SecretType.LM_HASH:
-        return f"{credentials.secret}:00000000000000000000000000000000"
+        return f"{plaintext_secret}:00000000000000000000000000000000"
 
     if credentials.secret_type == SecretType.NT_HASH:
-        return f"00000000000000000000000000000000:{credentials.secret}"
+        return f"00000000000000000000000000000000:{plaintext_secret}"
```

Comment:
For readibility, consider (in UTs as well):
```suggestion
        return f"{'0'*32}:{plaintext_secret}"
```

Reply:
This is data, not code. The less logic here, the more readable it is.

Reply:
This barely counts as logic. 😅 I don't feel too strongly about it, though. Feel free to skip.